tensorflow/compiler/mlir/tools/kernel_gen/kernel_creator.cc

```diff
@@ -389,7 +391,9 @@ Status LowerLoopsToGPUorCPU(mlir::ModuleOp module, bool embed_memref_prints,
   // Take launches to launches with kernels.
   if (!cpu_codegen) {
     const std::string gpuDataLayoutSpec =
-        "#dlti.dl_spec<#dlti.dl_entry<index,32:i32>>";
+        jit_i64_indexed_for_large_tensors
```

Comment:
At this point, you enforce i64 indexing (independently of tensor size). 
The better name would be `index_64bit`

Reply:
name changed.